common/middleware/set-services.test.js

```diff
@@ -0,0 +1,57 @@
+const proxyquire = require('proxyquire').noCallThru()
+
+const services = {
+  foo: {
```

Comment:
I find that in test code using metasyntactic variables like foo,bar,baz to be less readable than variables that describe what they mean or do.

For example here, you have a `serviceWithContext` and `serviceWithoutContext`